automated install/basic-install.sh

```diff
@@ -1652,7 +1652,7 @@ install_dependent_packages() {
         fi
     done
     if [[ "${#installArray[@]}" -gt 0 ]]; then
-        "${PKG_INSTALL[@]}" "${installArray[@]}" &> /dev/null
```

Comment:
Not sure if this is wanted, since this is a **yum** install, not an `apt-get` install! Not sure if yum can hang for the same reason, when redirecting STDOUT, but in all other yum calls `&> /dev/null` is still in place, hence this makes it inconsistent. I suggest to revert it in this PR. What we've done for apt-get could be done for yum as well (if required for same reason), but should be a dedicated PR then, where I would change `PKG_INSTALL` to be a function to handle apt-get + yum/dnf all together in a consistent way.

Reply:
If I was to keep this /dev/null in place, there would be no output between the horizontal rule on yum/dnf distributions. As for the redirections in other locations...I think they are fine for now.  
If we decide to remove them, those changes would belong in a different PR.